src/protocolgame.cpp

```diff
@@ -3593,21 +3595,21 @@ void ProtocolGame::AddPlayerSkills(NetworkMessage& msg)
 		msg.add<uint16_t>(0);                                                   // base special skill
 	}
 
-	// fatal, dodge, momentum
-	msg.add<uint16_t>(0);
-	msg.add<uint16_t>(0);
-
-	msg.add<uint16_t>(0);
-	msg.add<uint16_t>(0);
+	msg.addByte(0); // element magic level
+	// structure:
+	// u8 client element id
+	// u16 bonus element ml
 
-	msg.add<uint16_t>(0);
-	msg.add<uint16_t>(0);
+	// fatal, dodge, momentum
+	for (int i = 0; i < 3; ++i) {
+		msg.add<uint16_t>(0);
+		msg.add<uint16_t>(0);
+	}
 
 	// to do: bonus cap
-	msg.add<uint32_t>(player->hasFlag(PlayerFlag_HasInfiniteCapacity) ? 1000000
-	                                                                  : player->getCapacity()); // base + bonus capacity
-	msg.add<uint32_t>(player->hasFlag(PlayerFlag_HasInfiniteCapacity) ? 1000000
-	                                                                  : player->getCapacity()); // base capacity
+	uint32_t capacityValue = player->hasFlag(PlayerFlag_HasInfiniteCapacity) ? 1000000 : player->getCapacity();
```

Comment:
```suggestion
	uint32_t capacityValue = player->hasFlag(PlayerFlag_HasInfiniteCapacity) ? std::numeric_limits<uint32_t>::max() : player->getCapacity();
```

Reply:
This is visual, we want this value to be displayed, as it always has been.

Reply:
if we don't cap it to 10k for gm characters, the client will display 273489820394820348

Reply:
More of a client bug but OK

Reply:
Do you want any more or can it be merged?